common/persistence/src/main/java/app/coronawarn/server/common/persistence/service/DiagnosisKeyService.java

```diff
@@ -133,9 +133,9 @@ public void applyRetentionPolicy(int daysToRetain, String countryCode) {
         .ofInstant(Instant.now(), UTC)
         .minusDays(daysToRetain)
         .toEpochSecond(UTC) / SECONDS_PER_HOUR;
-    int numberOfDeletions = keyRepository.countOlderThan(threshold, countryCode);
+    int numberOfDeletions = keyRepository.countOlderThanOrEqual(threshold, countryCode);
```

Comment:
What is the reason we are falling back to `countOlderThanOrEqual`? We needed to change this method to `countOlderThan` in PR #711 to fix issue #700.